taxi/models.py

```diff
@@ -0,0 +1,26 @@
+from django.db import models
+
+from django.contrib.auth.models import AbstractUser
+
+from taxi_service import settings
+
+
+class Manufacturer(models.Model):
+    name = models.CharField(max_length=255, unique=True)
+    country = models.CharField(max_length=255)
+
+    def __str__(self):
+        return self.name
+
+
+class Driver(AbstractUser):
+    license_number = models.CharField(max_length=255, unique=True)
+
+
+class Car(models.Model):
+    model = models.CharField(max_length=255)
+    manufacturer = models.ForeignKey(Manufacturer, on_delete=models.CASCADE)

```

Comment:
Set related_name for manufacturer and drivers.

Reply:
yep)